contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -113,14 +113,16 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
 
         // Contract creations are signalled by sending a transaction to the zero address.
         if (decodedTx.to == address(0)) {
-            address created = Lib_SafeExecutionManagerWrapper.safeCREATE(
+            (address created, bytes memory revertData) = Lib_SafeExecutionManagerWrapper.safeCREATE(
                 decodedTx.gasLimit,
                 decodedTx.data
             );
 
             // EVM doesn't tell us whether a contract creation failed, even if it reverted during
             // initialization. Always return `true` for our success value here.
-            return (true, abi.encode(created));
+            return ( created == address(0) ) ?
```

Comment:
I think better to make this an if statement. The ternary syntax is a bit confusing here with the tuple return values.